src/content/docs/tutorial-reporting-data/capture-the-right-data.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: "Capture the right data with New Relic" 
+metaDescription: "This guide helps you set up the best telemetry possible to describe the runtime operation of your services."
+redirects:
+  - /docs/apm/agents/manage-apm-agents/agent-data/custom-instrumentation
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/sc-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/service-characterization-optimize-telemetry-data-guide/
+---
+
+Modern monitoring systems can give you important insight into the details of a service when they report the right data. Distributed trace, byte code, or script instrumentation allows teams to quickly collect detailed service telemetry. Unfortunately, operations teams usually aren't in the best position to evaluate the quality of the telemetry they get.
```

Comment:
> Distributed trace, byte code, or script instrumentation allows

Seems a bit clunky to me. Not sure what `byte code` is, but maybe something like "Distributed tracing, byte coding(?), or script instrumentation allow..."

Reply:
I've reworded this opening to be hopefully less clunky. Bytecode is actually a thing, but we can just remove it if it's too niche to put into an intro.